miner_exporter.py

```diff
@@ -182,12 +194,13 @@ def stats(miner: MinerJSONRPC):
     if version is not None:
         VALIDATOR_VERSION.labels(my_label).info({'version':version})
 
-    if this_validator is not None:
-        LEDGER_PENALTY.labels('ledger_penalties', 'tenure', my_label).set(this_validator['tenure_penalty'])
-        LEDGER_PENALTY.labels('ledger_penalties', 'dkg', my_label).set(this_validator['dkg_penalty'])
-        LEDGER_PENALTY.labels('ledger_penalties', 'performance', my_label).set(this_validator['performance_penalty'])
-        LEDGER_PENALTY.labels('ledger_penalties', 'total', my_label).set(this_validator['total_penalty'])
-        BLOCKAGE.labels('last_heartbeat', my_label).set(this_validator['last_heartbeat'])
+    if penalty_ledger is not None:
+        for name, ledger_entry in penalty_ledger.items():
```

Comment:
Filter out where status != staked?

Reply:
Good idea, I'll do this in the dict comprehension right when I collect above.

Reply:
done.